.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Remove this file

Reply:
Oh, sorry. that must be auto created file from vscode. I will remove that.